appstudio-utils/util-scripts/lib/fetch/cluster.sh

```diff
@@ -54,13 +59,14 @@ save-policy-config() {
       return 1
     fi
   else
-    local non_blocking_data_file
-    non_blocking_data_file="$( json-data-file config policy non_blocking_checks)"
-    echo "$non_blocking_data" | jq > "${non_blocking_data_file}"
+    # Save the config data from the ECP
+    echo "$config_data" | jq '{"config": {"policy": . }}' > "${config_file}"
```

Comment:
We're only fetching the non-blocking configuration from EnterpriseContractPolicies custom resource above, so it needs to be nested under `config.policy.nonBlocking`:

```suggestion
    echo "$config_data" | jq '{"config": {"policy": {"non_blocking_checks": . }}}' > "${config_file}"
```

Reply:
I might have found that if my cluster actually had the ECP resource working.

Reply:
I realized I changed the var name based on a misunderstanding. Will change it back in next revision.